### PR TITLE
[coffeescript/id] Create coffeescript-id.html.markdown

### PR DESCRIPTION
**Pull request by rilut**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

**Comment by rilut** [comment]
(Please squash these commits when merging. Thank you!)

File: id-id/coffeescript-id.html.markdown [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#  sayuran = sayur_sayuran[_k];
+#  if (sayuran !== 'kemangi') {
+#    makan(sayuran);
+#  }
+#}
+```
+
+## Referensi tambahan
+
+- [Smooth CoffeeScript] (http://autotelicum.github.io/Smooth-CoffeeScript/)
```
Comment (vendethiel) [inline id-id/coffeescript-id.html.markdown]: please put an " (EN)" tag or use id-id links

Comment (rilut) [inline id-id/coffeescript-id.html.markdown]: Oh ok, I'll put the "EN"

**Comment by vendethiel** [comment]
Thank you! 💟

**Comment by rilut** [comment]
Thanks :smile:

**Pull request merged**